src_base/xevd_ipred.c

```diff
@@ -676,7 +676,7 @@ void xevd_ipred_uv_b(pel *src_le, pel *src_up, pel *src_ri, u16 avail_lr, pel *d
     }
 }
 
-int intra_mode_list[IPD_CNT] = {
+/*int intra_mode_list[IPD_CNT] = {
```

Comment:
Can you remove this "intra_mode_list" from here instead of commenting?

Reply:
I remove "intra_mode_list".